ゴールが新規登録できないバグ

HumanCompilerにおいて、新規にゴールを追加しようとすると以下のバグが発生します。解消してください。

Failed to load resource: the server responded with a status of 403 ()Understand this error
common-13bb1806fec31ba0.js:1 🔍 [ApiClient] Response received: Object
vendor-b55219b9613727c8.js:6 [2025-08-27T22:39:20.012Z] [ERROR] Application Error: Not authenticated ApiError: Not authenticated
    at i.request (common-13bb1806fec31ba0.js:1:49111) Object
window.console.error @ vendor-b55219b9613727c8.js:6Understand this error
vendor-b55219b9613727c8.js:6 [2025-08-27T22:39:20.012Z] [ERROR][GoalFormDialog][createGoal] Failed to create goal ApiError: Not authenticated
    at i.request (common-13bb1806fec31ba0.js:1:49111) Object

--- a/apps/api/src/humancompiler_api/config.py
+++ b/apps/api/src/humancompiler_api/config.py
@@ -130,6 +130,7 @@ class Settings(BaseSettings):
                 expanded_origins.extend(
                     [
                         "https://humancompiler.vercel.app",
+                        "https://human-compiler.vercel.app",  # Add hyphenated domain
                         "https://humancompiler-five.vercel.app",
                         # Allow humancompiler prefixed domains only for security
                     ]

--- a/apps/api/src/humancompiler_api/routers/goals.py
+++ b/apps/api/src/humancompiler_api/routers/goals.py
@@ -31,6 +31,15 @@ def get_session():
         404: {"model": ErrorResponse, "description": "Project not found"},
     },
 )
+@router.post(
+    "",  # Handle requests without trailing slash
+    response_model=GoalResponse,
+    status_code=status.HTTP_201_CREATED,
+    responses={
+        404: {"model": ErrorResponse, "description": "Project not found"},
+    },
+    include_in_schema=False,  # Don't duplicate in OpenAPI schema
+)
 async def create_goal(
     goal_data: GoalCreate,
     session: Annotated[Session, Depends(get_session)],
@@ -69,6 +78,14 @@ async def get_goals_by_project(
         404: {"model": ErrorResponse, "description": "Goal not found"},
     },
 )
+@router.get(
+    "/{goal_id}/",  # Handle requests with trailing slash
+    response_model=GoalResponse,
+    responses={
+        404: {"model": ErrorResponse, "description": "Goal not found"},
+    },
+    include_in_schema=False,  # Don't duplicate in OpenAPI schema
+)
 async def get_goal(
     goal_id: str,
     session: Annotated[Session, Depends(get_session)],
@@ -95,6 +112,15 @@ async def get_goal(
         404: {"model": ErrorResponse, "description": "Goal not found"},
         422: {"model": ErrorResponse, "description": "Invalid status transition"},
     },
+)
+@router.put(
+    "/{goal_id}/",  # Handle requests with trailing slash
+    response_model=GoalResponse,
+    responses={
+        404: {"model": ErrorResponse, "description": "Goal not found"},
+        422: {"model": ErrorResponse, "description": "Invalid status transition"},
+    },
+    include_in_schema=False,  # Don't duplicate in OpenAPI schema
 )
 async def update_goal(
     goal_id: str,
@@ -147,6 +173,14 @@ async def update_goal(
     responses={
         404: {"model": ErrorResponse, "description": "Goal not found"},
     },
+)
+@router.delete(
+    "/{goal_id}/",  # Handle requests with trailing slash
+    status_code=status.HTTP_204_NO_CONTENT,
+    responses={
+        404: {"model": ErrorResponse, "description": "Goal not found"},
+    },
+    include_in_schema=False,  # Don't duplicate in OpenAPI schema
 )
 async def delete_goal(
     goal_id: str,

